hoot-services/src/main/java/hoot/services/controllers/info/MapInfoResource.java

```diff
@@ -117,16 +83,25 @@ public Response getMapSizes(@QueryParam("mapid") String mapIds) {
         JSONArray layers = new JSONArray();
 
         try {
-            String[] mapids = mapIds.split(",");
-            for (String mapId : mapids) {
+            List<Long> idList = new ArrayList<>();
+            if(mapIds == null) {
+                idList = DbUtils.getAllMapIds();
```

Comment:
We want to get map ids that the user can see, not all map ids in the db.

Maybe we just want to move the size computation into `MapResource.getLayers()` and eliminate the need for a second request.

Reply:
yea im going to merge these 2...also seems more efficient to have them in 1 request since currently its always calling 1 after the other